include/ignition/gazebo/detail/EntityComponentManager.hh

```diff
@@ -395,7 +440,7 @@ void EntityComponentManager::Each(typename identity<std::function<
 //////////////////////////////////////////////////
 template<typename ...ComponentTypeTs>
 void EntityComponentManager::Each(typename identity<std::function<
-    bool(const Entity &_entity, ComponentTypeTs *...)>>::type _f)
+    bool(const Entity &, ComponentTypeTs *...)>>::type _f)
```

Comment:
Is this change needed? I don't see it made for other `Each*` method signatures.

Reply:
Reverted in https://github.com/ignitionrobotics/ign-gazebo/pull/1317/commits/48b2befd4efaed475c632058ae1783193eecdd55. In a previous version, I think I was making changes to the signatures and thought I should use the opportunity to remove the unused argument name (clang-tidy gives a warning in my editor).